pkg/pf/tests/internal/cross-tests/configure.go

```diff
@@ -109,7 +109,8 @@ func Configure(t T, schema schema.Schema, tfConfig map[string]cty.Value, options
 	// Run the TF part
 	{
 		var hcl bytes.Buffer
-		err := writeProvider(&hcl, schema, providerName, tfConfig)
+		sch := NewHCLSchemaPFProvider(schema)
+		err := crosstestsimpl.WriteProvider(&hcl, sch, providerName, tfConfig)
```

Comment:
I don't see a reason to introduce intermediary state here.
```suggestion
		err := crosstestsimpl.WriteProvider(&hcl, NewHCLSchemaPFProvider(schema)
, providerName, tfConfig)
```